xbmc/filesystem/CurlFile.h

```diff
@@ -24,11 +24,17 @@
 #include <map>
 #include <string>
 #include "utils/HttpHeader.h"
+#include <curl/curlver.h>
 
 namespace XCURL
 {
+#if LIBCURL_VERSION_MAJOR > 7 || (LIBCURL_VERSION_MAJOR == 7 && LIBCURL_VERSION_MINOR >= 50)
```

Comment:
#if LIBCURL_VERSION_NUM >= 0x073200 // 7.50.0

sorry, I missed this in the header, I thought  (facepalm) it was not updated. but it is ok